janusgraph-test/src/test/java/org/janusgraph/graphdb/tinkerpop/JanusGraphIoRegistryTest.java

```diff
@@ -60,7 +60,7 @@ public void testTinkerPopPredicatesAsGryo() throws SerializationException {
         assertEquals(15,
                 Stream.of(P.class.getDeclaredMethods()).filter(m -> Modifier.isStatic(m.getModifiers())).filter(p -> {
                     log.debug("Predicate: {}", p);
-                    return true;
+                    return !p.isSynthetic();
```

Comment:
Intentional change? It's unclear how this follows from the rest of the PR, and the comment seems ominous. I'm not sure how to interpret this. I'll add some more folks to review this and `pom.xml` changes as well.